src/delayP.js

```diff
@@ -0,0 +1,39 @@
+import { curry, propOr, partial, nth } from 'ramda';
+
+import isPlainObj from './isPlainObj';
+/**
+ *
+ * @func delayP
+ * @memberOf RA
+ * @category Function
+ * @sig a -> Promise a
+ * @param {Number|Object} Argument to be resolved after a certain timeout and returns value if provided.
+ * @return {Promise} A Promise that is resolved with the given value if provided after certain delay.
+ * @see {@link RA.delayP|delayP}
+ * @example
+ *
```

Comment:
In examples we're trying to describe the result value, not how we got to that value

Reply:
Okay :)